### ISSUE TITLE
Feature Request: Create an Examples Landing page

### ISSUE THREAD

**Issue by johnwalicki** [open-horizon/open-horizon.github.io#372]
### Is your feature request related to a problem? Please describe.

The Open Horizon documentation should include an Examples landing page that guides developers to the [Examples repo](https://github.com/open-horizon/examples) and the [Open Horizon Services repos](https://github.com/orgs/open-horizon-services/repositories).

It might be a sortable table with the example service name, link, short description, architecture.

This examples landing page should be linked to from the `Developing` card on the main OH page.

**Comment by johnwalicki** [comment]
We discussed this idea on the Examples working group call on 01/03/2023 as it relates to the sunset of the SDR documentation.  See #358

**Comment by joewxboy** [comment]
If possible, it would be nice for it to be programmatically generated.  Bonus points for using a GHA.

**Comment by johnwalicki** [comment]
@joewxboy You set the bar high :-)
I'm thinking about hand building the table the first time, iterating on what's useful. Then we can consider automation.

**Comment by joewxboy** [comment]
> @joewxboy You set the bar high :-) I'm thinking about hand building the table the first time, iterating on what's useful. Then we can consider automation.

Crawl, then walk, then run!  Sounds good.

**Comment by johnwalicki** [comment]
I posted a PR that builds an initial table of Examples.  Let's see what it looks like on the OH documentation website. 

I purposefully do not link the main index to it yet.

While the table looks impressive (and I didn't even include all of the examples in the examples repo), I'm not at all impressed with the quality of these examples.  Would grade many of them as a `C-` tutorials.

**Comment by johnwalicki** [comment]
It looks reasonably good.
https://open-horizon.github.io/docs/developing/examples.html

**Comment by joewxboy** [comment]
Q: Are there any pages linking to it?  I don't want it to be an orphan.

Thoughts for logical linking pages include:

- https://open-horizon.github.io/docs/ in "Developing" box, linked as "Learning from example edge services"
- https://open-horizon.github.io/docs/developing/ in list, linked as "List of all example edge services for both devices and clusters"
- https://open-horizon.github.io/docs/developing/further_reading.html in list, linked as "Community-contributed example edge services"

**Comment by johnwalicki** [comment]
Purposefully, this page is an orphan, until we deem it ready.  I though we might fill in some of the missing fields, add more of the services listed in the Examples repo.  I ran out of time in my initial commit. (Mostly because many of the examples don't even have a rudimentary one sentence description of what they do)

In my initial comment above, I suggested it be linked to from the main landing page.  I  proposed we add it to the Developing card.
